### PR TITLE
Add ability to set noobaa cleanup policy during uninstall

### PR DESCRIPTION
**Pull request by iamniting**
This code setup the noobaa cleanup policy to destroy the obc

[1]. https://github.com/noobaa/noobaa-operator/blob/v2.3.0/pkg/apis/noobaa/v1alpha1/noobaa_types.go#L331

### CODE REVIEW

**Comment by iamniting** [comment]
@raghavendra-talur @obnoxxx @jarrpa Can you pls review the PR and set `ok to test`

File: pkg/controller/storagecluster/reconcile.go [hunk 1]
```diff
@@ -502,6 +503,29 @@ func (r *ReconcileStorageCluster) setRookCleanupPolicy(instance *ocsv1.StorageCl
 	return nil
 }
 
+// set noobaa cleanup policy to destroy obc
+func (r *ReconcileStorageCluster) setNoobaaCleanupPolicy(sc *ocsv1.StorageCluster, reqLogger logr.Logger) error {
```
Comment (raghavendra-talur) [inline pkg/controller/storagecluster/reconcile.go:507]: Please have "Uninstall:" as a prefix for all the log lines.

Also, have a log line that prints a success case, may be at line 527.

Comment (iamniting) [reply pkg/controller/storagecluster/reconcile.go:507]: Done

**Comment by iamniting** [comment]
> Please merge the two commits. In general, don't split the function definition and it's call.

Done

**Pull request merged**